scrapy/pqueues.py

```diff
@@ -110,6 +111,28 @@ def __len__(self):
         return sum(len(x) for x in self.queues.values()) if self.queues else 0
 
 
+class ScrapyDelayedRequestsPriorityQueue(ScrapyPriorityQueue):
+    """This priority queue has the similar implementation as default ScrapyPriorityQueue
+    but it uses timestamps as priority. It allows to easily add per request delays and
+    doesn't touch the default queue.
+    If this priority queue doesn't contain any requests to process or the current time is less than
+    the next time when request should be processed it returns None and request from the main priority queue
+    will be taken.
+    """
+
+    def priority(self, request):
+        now = int(datetime.now().timestamp())
```

Comment:
:lipstick: I wonder why you did not go for `time.time()`, is there a relevant functional difference, or was it just personal preference?

Reply:
Don't have any specific preferences. Replaced to `time.time()` in terms of shortening.